Source/Extensions/Blazorise.DataGrid/_DataGridRow.razor

```diff
@@ -64,6 +66,15 @@
                     }
                 </TableRowCell>
             }
+        }        
+        @if (ParentDataGrid.OverlayTemplate != null && mouseIsOver)
+        {
+            <TableRowCell Style="width:0px;margin:0px;padding:0px" Overflow="Overflow.Visible">
+                <Div Style="position:absolute;right:0px;margin:0px;padding;0px">

```

Comment:
Prefer Blazorise utilities over inline styles

- Position
- Margin
- Padding

Reply:
Changed to use utilities